src/main/java/org/example/commands/DeleteDependencyCommand.java

```diff
@@ -2,39 +2,56 @@
 
 import java.util.ArrayList;
 import java.util.Map;
-import java.util.Scanner;
 import org.example.exceptions.InvalidArgument;
 import org.example.model.Node;
+import org.example.services.ConsoleInputServiceInterface;
 
 public class DeleteDependencyCommand implements CommandInterface {
+  private final ConsoleInputServiceInterface consoleInputService;
+
+  public DeleteDependencyCommand(final ConsoleInputServiceInterface consoleInputService) {
```

Comment:
you have removed this final in next prs can I know why?

Reply:
There were some issues with the scanner , added final again